constants/Onboarding.js

```diff
@@ -0,0 +1,29 @@
+export const ONBOARDING_CONTENT = [
+  {
+    title: 'Find stores for you',
+    body:
+      'Explore the map to discover nearby stores stocking healthy fruits and vegetables',
+  },
+  {
+    title: 'Know what’s in stock',
+    body:
+      'See what products are available in stores before you even leave the house',
+  },
+  {
+    title: 'Shop healthy and save',
+    body:
+      'Sign up for Healthy Rewards to earn $5 rewards when you buy healthy products',
+  },
+  {
+    title: 'Stay informed',
+    body:
+      'Access our resource database to help you continue eating and living healthy',
+  },
+];
+
+export const ONBOARDING_IMAGES = [
+  require('../assets/images/search_map-marker-alt_apple-alt.png'),
+  require('../assets/images/home_shopping-basket_pepper-hot.png'),
+  require('../assets/images/star_money-bill_carrot.png'),
+  require('../assets/images/heart_balance-scale_user.png'),
+];
```

Comment:
Should we get these from Airtable since they're only used once?

Reply:
I feel they should be in the static folder because they are constant + pulled once, and whose space is not very large too! Want to reduce the amount of things to pull from airtable if possible in case network is an issue haha